### ISSUE TITLE
config.json read from the wrong path

### ISSUE THREAD

**Issue by semafor** [FriskByBergen/python-friskby#9]
Testing out the image and python-friskby 0.62.0, I get the following output after successfully downloading a config to `/usr/local/friskby/etc/config.json`

https://paste.debian.net/plain/924848

Is python-friskby using the wrong path?

**Comment by pgdr** [comment]
Yes, it seems to be looking for /usr/local/etc/config.json (note no friskby).
